code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -157,15 +164,19 @@
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/assembly/signaler(src) // 0 tc
 			new /obj/item/storage/toolbox/syndicate(src) // 1 tc
-
+			new /obj/item/pen/edagger(src)
+			new /obj/item/gun/energy/wormhole_projector(src) //mooorttyyyy
```

Comment:
do we need this reference

Reply:
I can make the portals green if you like

Reply:
Regardless its “sciencey” and its mad scientist bundle and honesty its such a shitty bundle even with these changes im wondering if people will still hate it, im just trying to buff it while staying with the theme